### DISCUSSION TITLE
AOSP 11.0 v313

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2026] [General]
- October security patch
- 313!

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v313'>AOSP 11.0 v313</a>.</em>

**Comment by hwti** [comment]
**October** security patch, no ?

**Reply by phhusson** [reply]
yup thanks

**Reply by innofeld** [reply]
Dear phhusson @phhusson
Special thanks
Please use LTS kernels in your releases to obtain best possible results
> https://android.googlesource.com/kernel/common/+/refs/heads/android11-5.4-lts
This kernel is really perfect, all bugs fixed and highly trusted.
Regards
INNOFELD

**Comment by seyhanmahmut** [comment]
Aosp 12 huawei p20 lite???

**Reply by jsjsowo** [reply]
hhh

**Comment by ghost** [comment]
Didn't know Duck Tales was still a thing... lol

Thanks for the release!

**Comment by innofeld** [comment]
Dear phhusson @phhusson 
Special thanks
Please use LTS kernels in your releases to obtain best possible results
    >  https://android.googlesource.com/kernel/common/+/refs/heads/android11-5.4-lts
This kernel is really perfect, all bugs fixed and highly trusted.
Regards
INNOFELD

**Reply by vatva691** [reply]
stop spam those kernel, it's not working on treble device

**Reply by vatva691** [reply]
treble device only enable generic system image, they are not related into kernel, you must wait gki on future device, but not now.

**Reply by innofeld** [reply]
Understood
So, I need to know, what's latest kernel on this existing Treble image which released by 313 ?

**Reply by vatva691** [reply]
well, it's depend on your stock kernel, kernel verison is same on your latest stock rom, gsi rom only change system.

**Reply by innofeld** [reply]
Well,
I have a smartphone to test and evaluation, smartphone is BV9500-plus from Blackview
It has Android 9 Pie
So do you know an effective procedure to install AOSP 313 on my smartphone?
Regards

**Comment by Miustone** [comment]
Hail Ducktales hahahaha <3

**Comment by JonnyTech** [comment]
Loving the boot logo! 8D

**Comment by Miustone** [comment]
Am i the only One who has this "Device not certified" Issue? Xperia 5 and pretty annyoing these unlimited Notfications. Will Magisk be fixed on Android 12 btw? It's working fine for Me on Beta 5 GSI so i am wondering... dear @phhusson <3

**Comment by ManasesGit** [comment]
Hi Phhusson,

Thanks for a lovely ROM. This is almost perfect. My only challenge is a small bug with Chrome.

I am using Infinix x690B with the arm64 A/B Gapps rom. For some strange reason, my Bluetooth headset is able to connect but unable to stream media audio, only phone calls. This happens only when I update the Chrome app to the latest version on the play store.

I have checked the Force headset switch, use alternate way to detect, disabled audio effects, mediatek on the Bluetooth workarounds and force disable A2DP offload. This help remove echoes during a call but not the Bluetooth issue.

I am sure this has to do with Chrome and thought you should know.

Thanks once again for all the great work.

**Reply by amosayomide05** [reply]
Hey, please how do I know which file to download

**Reply by GospelUgo** [reply]
Please did you download gapps separately or it came with the rom, my phone is X690, I hope it will work in mine?

**Comment by jj573702** [comment]
Thank you for your work.

**Reply by XenonTheInertG** [reply]
Thankssss

**Comment by AndroiDFlasher** [comment]
Tryed to flash system-roar-arm64-aonly-floss, which i extracted to img file on my Oukitel C11 Pro device. Get image file larger than targeted device error.
When I flash vanilla version, os boots but can t pass lockscreen, glitches. Device MTK 6739

**Comment by lalakii** [comment]
Dear phhusson @phhusson !  Good ! Good ! Good !

**Comment by DPs18** [comment]
Hey @phhusson I just wanna ask which one will I download for my infinix hot 10 play? Your reply will be much appreciated. Thank you!

**Reply by TechySkills** [reply]
> Hey @phhusson I just wanna ask which one will I download for my infinix hot 10 play? Your reply will be much appreciated. Thank you!

https://www.getdroidtips.com/android-11-infinix-hot-10-play-x688c/

**Reply by TechySkills** [reply]
> Hey @phhusson I just wanna ask which one will I download for my infinix hot 10 play? Your reply will be much appreciated. Thank you!

I think, this would work.

**Reply by TechySkills** [reply]
> Hey @phhusson I just wanna ask which one will I download for my infinix hot 10 play? Your reply will be much appreciated. Thank you!

Your device is suitable for Lineage OS 18.1. So, you can download this, according to your partition. a/ab:
https://sourceforge.net/projects/amber-gsi/files/LineageOS-18.1/

Download Gapps (Google Apps) if you want from this link:
https://sourceforge.net/projects/opengapps/files/arm64/20211117/open_gapps-arm64-11.0-stock-20211117.zip/download?use_mirror=webwerks&r=&use_mirror=autoselect

**Reply by TechySkills** [reply]
> Hey @phhusson I just wanna ask which one will I download for my infinix hot 10 play? Your reply will be much appreciated. Thank you!

For TWRP, You can try out this link:
https://www.getdroidtips.com/twrp-recovery-infinix-hot-10/

**Comment by tosuman** [comment]
please fix the JIO volte issue in exynos processor 9609/9610 please My phone is motorola one action

**Comment by filevest** [comment]
Settings keep crashing upon setting Date please fix this issue on arm64-gsi-gapps

**Comment by 124sami** [comment]
Can i install this gsi in my device ??? Please anyone help me . 🙏🏼
Device details :
Name : Walton Primo H8 Turbo
Android Version : 8.1
Processor : MT6739
Ram : 3 GB

**Reply by devrieshh** [reply]
install the treble check app from playstore ,you will see what gsi you can use

**Reply by innofeld** [reply]
Dear @devrieshh 
in my phone's treble check app, it shows "VNDK 28"
So what it means? and can I install this 313 version?
Regrads

**Comment by devrieshh** [comment]
Does the app says that your phone have system as root? Than you can install
a/b ROM,other wise,only a only roms

**Comment by devrieshh** [comment]
28 means android 8.1 android 12 ,is about 31

**Comment by devrieshh** [comment]
Also check if your phone is 32bit or 64

**Reply by innofeld** [reply]
Dear @devrieshh 
My smartphone is Treble enabled and VNDK 28
It's ARM64, 
So, can I install AOSP 11.0 v313 or AOSP 12.0x ?
Thanks and Regards
INNOFELD

**Reply by 124sami** [reply]
> Also check if your phone is 32bit or 64

my phone is 32 bit a only

**Comment by devrieshh** [comment]
Yes,the ,Aonly for Shure a/b only when your phone support system as root
see treble app

**Comment by devrieshh** [comment]
So 11.0 v313 can

**Reply by innofeld** [reply]
Dear @devrieshh
Thanks for guide, my smartphone is Blackview BV9500-plus.
Some of the sources refer A-Only, and some of them A/B
I guessed it may be A/B,
what do you think?
Regards

**Comment by innofeld** [comment]
Dear @devrieshh 
Thanks for guide, my smartphone is Blackview BV9500-plus.
Some of the sources refer A-Only, and some of them A/B
I guessed it may be A/B, 
what do you think?
Regards

**Reply by JonnyTech** [reply]
Use this app to check: https://play.google.com/store/apps/details?id=tk.hack5.treblecheck

**Comment by devrieshh** [comment]
When it does support system as root /
You can take a/b

**Reply by innofeld** [reply]
Thanks a lot for your valuable time!

what's your opinion about ab-vndklite-vanilla.img.xz ?
I heard vanilla is a clear and pure-android, 
Do you think we should use Gapps or Floss versions?

Regards

**Comment by devrieshh** [comment]
Gapps is with Google,floss with alternative apps,with vanilla the system is
pure, sometimes things of a phone work only with vanilla

**Comment by innofeld** [comment]
Dear @phhusson 

Please pay attention to 11.0.0_r51 (RSV1.210329.035) / January 4, 2022; 28 days ago
Latest security patch on this ROM

And in continuation, please inform us if we can use this ROM v313 with secured variant.
My smartphone is BV9500-plus.  

Many thanks 

Best Regards

@innofeld

**Comment by apbucaneg** [comment]
since A11 release, no more GSI for arm32-binder64-aonly? looking for them in A11 releases and i can't find them. thanks in advance.

**Comment by kyldpr** [comment]
Where is the boot animation zip file located

**Comment by JonnyTech** [comment]
The animation is built into the framework, there is no discrete file.

**Comment by pawcio950** [comment]
Hello I need android 11 with no su and no vndk for arm32_binder64, can I build it from this source?

**Reply by devrieshh** [reply]
https://github.com/phhusson/treble_experimentations/releases/tag/v313 you have only to uninstall su
